demo.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from enum import IntEnum
+
+from telegram import ReplyKeyboardMarkup, ReplyKeyboardRemove
+from telegram.ext import CommandHandler, MessageHandler, Filters, ConversationHandler
+
+
+class DemoDecisions(IntEnum):
+    FAVPIC = 1
+    FAMILY = 2
+    AMAZON = 3
+    DURABLE = 4
+    DIFFICULT = 5
+    MICHAEL = 6
+    WINNER = 7
+
+
+yes_no_keyboard = ReplyKeyboardMarkup([["Yes", "No"]])
+filter_yes = Filters.regex("^Yes$")
+filter_no = Filters.regex("^No$")
+
+
+def demo_intro(update, context):
+    text = "Send your favorite picture of Covid!"
+    update.message.reply_text(text=text, reply_markup=ReplyKeyboardRemove())
+    return DemoDecisions.FAVPIC
+
+
+def demo_family(update, context):
+    text = "Do you agree Covid-19 does have neither family or friends?"
+    update.message.reply_text(text=text, reply_markup=yes_no_keyboard)
+    return DemoDecisions.FAMILY
+
+
+def demo_family_invalid_input(update, context):
+    text = "Please send me a picture!"
+    update.message.reply_text(text=text)
+
+
+def demo_amazon(update, context):
+    text = "How likely would you give Covid-19 0 stars on Amazon? (10=most likely)"
+    keyboard = ReplyKeyboardMarkup([["1", "2", "3"], ["4", "5", "6"], ["7", "8", "9"], ["10"]])
+    update.message.reply_text(text=text, reply_markup=keyboard)
+    return DemoDecisions.AMAZON
+
+
+def demo_durable(update, context):
+    text = "Mohammed Ali or Covid-19, who's more durable?"
+    keyboard = ReplyKeyboardMarkup([["Mohammed Ali", "Covid-19"]])
+    update.message.reply_text(text=text, reply_markup=keyboard)
+    return DemoDecisions.DURABLE
+
+
+def demo_difficult(update, context):
+    text = "What's more difficult do build?"
+    keyboard = ReplyKeyboardMarkup([["Chinese Wall", "Golden-Gate-Bridge"], ["Eiffel Tower", "Colosseum", "Covid-19 vaccine"]])
+    update.message.reply_text(text=text, reply_markup=keyboard)
+    return DemoDecisions.DIFFICULT
+
+
+def demo_michael(update, context):
+    text = "If Michael was alive, would he know how to heal Covid-19?"
+    keyboard = ReplyKeyboardMarkup([["Yes", "No", "Not so sure"]])
+    update.message.reply_text(text=text, reply_markup=keyboard)
+    return DemoDecisions.MICHAEL
+
+
+def demo_winner(update, context):
+    text = "Who wins eventually?"
```

Comment:
```suggestion
    text = "Who will win eventually?"
```
future. Alternative: going to win

Reply:
```suggestion
    text = "Who is going to win eventually?"
```